### PR TITLE
Add static linking aPLib detection

### PR DESCRIPTION
**Pull request by Still34**

### CODE REVIEW

File: linking/static/aplib/linked-against-aplib.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  meta:
+    name: linked against aPLib
+    namespace: linking/static/aplib
+    authors:
+      - still@teamt5.org
+    scope: file
+    mbc:
+      - Data::Compression Library [C0060]
+    examples:
+      - 72c8e3c8049927fa1dd53b61d9b67b9d74ab5b6c030bc38dfbe9a338a32438c9
```
Comment (mr-tz) [inline linking/static/aplib/linked-against-aplib.yml:11]: can you upload this to the capa-testfiles repo, please?

Comment (Still34) [reply linking/static/aplib/linked-against-aplib.yml:11]: Done

**Review by mr-tz — APPROVED** [review]
lgtm, thanks

**Review by williballenthin — APPROVED** [review]
thank you!

**Pull request merged**